3-networks/modules/dedicated_interconnect/variables.tf

```diff
@@ -103,3 +103,9 @@ variable "region2_router2_name" {
   type        = string
   description = "Name of the Router 2 for Region 2 where the attachment resides"
 }
+
+variable "cloud_router_labels" {
+  type        = map(string)
+  description = "A map of suffixes for labelling vlans"
```

Comment:
nit: can we add what the structure should be? for example`"vlan_1" =>"suffix1"`, so the use knows what are the right keys. If not in the description, you can also add a comment in the code as well, just so its easier for the end user.

Reply:
Done